src/Neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -458,6 +458,7 @@ public virtual VerifyResult VerifyStateIndependent(ProtocolSettings settings)
 
         public StackItem ToStackItem(ReferenceCounter referenceCounter)
         {
+            if (_signers == null || _signers.Length == 0) throw new ArgumentException("Sender is not specified in the transaction.");
```

Comment:
Will this change fork something?

Reply:
No, it will still fault. Originally it fault, now it still fault, but fault with different exception, a more reasonable exception message.